nimare/meta/utils.py

```diff
@@ -359,19 +360,33 @@ def compute_kda_ma(
         idx = (np.min(sphere, 1) >= 0) & (np.max(np.subtract(sphere, shape), 1) <= -1)
         sphere = sphere[idx, :].astype(int)
         exp = exp_idx[i]
-        if sum_overlap:
-            kernel_data[exp][tuple(sphere.T)] += value
+        if memmap_filename:
```

Comment:
If we're going to start using sparse arrays, should we just drop memory-mapping entirely?

Reply:
I was leaning that to be another PR since that could be a decent amount of work, maybe the removal could be a PR to this PR?

Reply:
I'm fine tackling that separately. I'm just glad to know that we're planning to simplify things eventually.